src/Storage/Program.cs

```diff
@@ -189,12 +190,13 @@ void ConfigureServices(IServiceCollection services, IConfiguration config)
     services.AddHealthChecks().AddCheck<HealthCheck>("storage_health_check");
 
     services.AddHttpClient<AuthorizationApiClient>();
+    services.AddHttpClient<IRegisterService, RegisterService>();
 
     services.Configure<AzureStorageConfiguration>(config.GetSection("AzureStorageConfiguration"));
     services.Configure<GeneralSettings>(config.GetSection("GeneralSettings"));
     services.Configure<KeyVaultSettings>(config.GetSection("kvSetting"));
     services.Configure<PepSettings>(config.GetSection("PepSettings"));
-    services.Configure<PlatformSettings>(config.GetSection("PlatformSettings"));
+    services.Configure<Altinn.Platform.Storage.Configuration.PlatformSettings>(config.GetSection("PlatformSettings"));
```

Comment:
@codemascot Here we should have added a new line instead of changing the one we had. We can also rename the new type from PlatformSettings to RegisterClientSettings to avoid the ambiguity.

Reply:
Correct! 👍 Here is the implementation commit: https://github.com/Altinn/altinn-storage/pull/376/commits/25aba89a59880ff8632d11269a6d696d6cb0be04 🙂